app/api/made_to_measure/subscribers.rb

```diff
@@ -0,0 +1,48 @@
+module MadeToMeasure
+
```

Comment:
ja, he likes spaces. I kinda like no spaces in between things. 2 against 3 :wink: